Resources/ServerInfo/Guidebook/ServerRules/SiliconRules/RuleS8DefaultCrewDefinition.xml

```diff
@@ -1,4 +1,15 @@
 ﻿<Document>
   # Silicon Rule 8 - Your HUD determines who is crew
   Unless a law redefines the definition of crew, then anyone who the HUD indicates to you has a job, including passengers, is a crewmember. You cannot do something that causes someone to not be considered crew, but you can allow someone else to do something that causes someone to not be crew.
+  
+  The following icons are not considered crew:
+  <Box>
+    <GuideEntityEmbed Entity="NoIdIcon" Caption="No ID"/>
+    <GuideEntityEmbed Entity="UnknownIcon" Caption="Unknown"/>
+    <GuideEntityEmbed Entity="VisitorIcon" Caption="Visitor"/>
+    <GuideEntityEmbed Entity="BorgIcon" Caption="Cyborg"/>
+    <GuideEntityEmbed Entity="StationAiIcon" Caption="Station AI"/>
+    <GuideEntityEmbed Entity="SyndicateIcon" Caption="Syndicate"/>
```

Comment:
As far as I know, this icon can also never be seen except for Agent ID. Same justification as zombie icon.

```suggestion
```

Reply:
Some people still get confused by seeing the syndicate ID, so i'll keep it in there.

Reply:
Yeah I've been convinced it's fine to keep as some syndicates may want it for role-play purposes (or for a funny thief to disguise as one).